src/Frontend/Themes/Bootstrap/src/Layout/Templates/Pagination.html.twig

```diff
@@ -39,7 +39,7 @@
           {% for pagination in pagination.last %}
             <li>
               <a href="{{ pagination.url }}" rel="nofollow" title="{{ 'lbl.GoToPage'|trans|capitalize }}{{ pagination.label }}">
```

Comment:
fixed in https://github.com/sumocoders/forkcms/pull/111/files#diff-69e31cd3d15a7a00cf3ad55686ab0d01